### PR TITLE
added solution of hw-5.2

### PR DESCRIPTION
**Pull request by KseniiaMakarova**

### CODE REVIEW

File: src/main/java/core/basesyntax/Main.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class Main {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Main.java]: please remove this class. you needed it to check yourself

File: src/main/java/core/basesyntax/Plane.java [hunk 2]
```diff
@@ -7,5 +7,58 @@
...
+    public static class PlaneBuilder {
+        private String model;
+        private int capacity;
+        private int weight;
+        private int maxSpeed;
+        private int noiseLevel;
+
+        public PlaneBuilder() {
+
+        }
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Plane.java]: do you need to create this constructor?

File: src/main/java/core/basesyntax/Main.java (hunk 1 above)
Comment (KseniiaMakarova) [reply src/main/java/core/basesyntax/Main.java]: sure!

File: src/main/java/core/basesyntax/Plane.java (hunk 2 above)
Comment (KseniiaMakarova) [reply src/main/java/core/basesyntax/Plane.java]: I agree, it was redundant
done)

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**